### ISSUE TITLE
[Notice] PyTorch version

### ISSUE THREAD

**Issue by akirasosa** [klicperajo/dimenet#4]
Hi

I would like to share that I have ported this repo to PyTorch version. Though it's during training still, I have seen quite good result.
https://github.com/akirasosa/pytorch-dimenet

Feel free to close this post, as this is just a notice.

Thanks for great paper,
Akira

**Comment by gasteigerjo** [comment]
Nice, well done! I can spot some differences in the weight initialization, optimizer and training schedule. I'm curious, please do let me know how it goes! :)

Also, we couldn't get half-precision to work properly, which seemed reasonable considering that the achieved accuracy of DimeNet is better than 16-bit machine precision for some targets. But maybe something else was the issue? If you can get this to work, I'd be very interested!
